Bricks/cerealize/cerealize.h

```diff
@@ -212,7 +204,7 @@ class CerealBinaryFileParser {
     if (fi_.peek() != std::char_traits<char>::eof()) {  // This is safe with any next byte in file.
       std::unique_ptr<T_ENTRY> entry;
       si_(entry);
-      processor(*entry.get());
+      processor(std::move(*entry.get()));
```

Comment:
True -- but it's still not `const`.

`std::move()` here is an indicator that the client is free to invalidate the object passed to it.

Taking away ownership is not the same as destructing. For instance, passing a `unique_ptr` to a polymorphic type by an rvalue reference enables `std::move()`-ing it further without copying. The destructor of the original `unique_ptr<>` defined two lines above will still be called, as it should to ensure no memory is leaked.